firefox-ios/Client/FeatureFlags/NimbusFlaggableFeature.swift

```diff
@@ -66,6 +66,10 @@ struct NimbusFlaggableFeature: HasNimbusSearchBar {
         case .jumpBackIn:
             return FlagKeys.JumpBackInSection
 
+        // Cases where users manipulate a setting via debug menu.
+        case .microsurvey:
```

Comment:
conversation we had re: features that require both user & debug flags and how the system should be able to handle both

Reply:
update in this commit, hope it addresses the concerns: 
f881c7e85f3dc66c092bc7076a518970908722a0